index.html

```diff
@@ -1034,13 +1038,19 @@ <h2 id="BaseAudioContext">
               <li>In case of failure, queue a task on the <a>control thread</a>
               to execute the following, and abort these steps
                 <ol>
+                  <li>Reject all promises from <a>pendingResumePromises</a> in
+                  order, then clear <a>pendingResumePromises</a>.
+                  </li>
                   <li>Reject <em>promise</em>
```

Comment:
This is a little confusing?  You've already added the promise to `pendingResumePromises` above because the context is not allowed to start.  So you reject the promise when handling `pendingResumePromises`, and then reject the promise again in this step.